src/compute/datomic_client_memdb/core.clj

```diff
@@ -68,6 +68,9 @@
       :t (peer/basis-t db)
       :next-t (inc (:t this))
       :db-name db-name
+      :as-of (peer/as-of-t db)
+      :since (peer/since-t db)
+      :history (boolean (or (peer/as-of-t db) (peer/since-t db)))
```

Comment:
There is no peer function to check if a db is historic or not, so I just guessed that if it has as-of or since => its historic.